### PR TITLE
Feature/styling

### LINKED ISSUE #46: add scroll to top function, so that, as a user, I see the top of the page every time I go to a new page even if I have scrolled on the previous page view

### LINKED ISSUE #48: add go back button to details card

### PR DESCRIPTION
**Pull request by clairefields15**
### Acceptance Criteria
Add styling to details page, including go back button. Add scroll to top so that users are brought to the top of each new page (not sure how to make it different for go forward and go back). 

### Issues Resolved
closes #48 
closes #46 

### Code Reviewers
@Meekb @alexmkio @AshleyOh-bit

### CODE REVIEW

File: src/scrollToTop.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { useEffect } from 'react';
+import { useLocation } from 'react-router-dom';
+
+export default function ScrollToTop() {
+  const { pathname } = useLocation();
```
Comment (AshleyOh-bit) [inline src/scrollToTop.tsx:5]: This is so clever! Great job using Router to its fullest extent.

File: src/scrollToTop.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+  useEffect(() => {
+    window.scrollTo(0, 0);
```
Comment (alexmkio) [inline src/scrollToTop.tsx:8]: interesting. Nice work!

**Review by alexmkio — APPROVED** [review]

File: src/App.tsx [hunk 3]
```diff
@@ -70,47 +71,51 @@ export const App: React.FC = () => {
   };
 
   return (
-    <main>
+    <>
```
Comment (Meekb) [inline src/App.tsx:74]: WHOA!!!!!! React fragment short-hand?!

**Review by Meekb — APPROVED** [review]

File: src/components/Details/Details.css [hunk 4]
```diff
@@ -1,23 +1,109 @@
...
+}
+
+@media screen and (min-width: 900px) {
+  .market-name {
+    font-size: 2em;
+  }
+}
+
+@media screen and (min-width: 1200px) {
+}
```
Comment (AshleyOh-bit) [inline src/components/Details/Details.css:109]: Is this changing anything right now?

**Review by AshleyOh-bit — APPROVED** [review]

File: src/components/Details/Details.css (hunk 4 above)
Comment (clairefields15) [reply src/components/Details/Details.css:109]: No, there are def extra media queries in every css file. I will remove as needed!

**Pull request merged**